clients/java/src/main/java/io/zeebe/client/impl/oauth/OAuthCredentialsProvider.java

```diff
@@ -72,9 +71,12 @@ public void applyCredentials(final Metadata headers) throws IOException {
       loadCredentials();
     }
 
-    headers.put(
-        HEADER_AUTH_KEY,
-        String.format("%s %s", credentials.getTokenType(), credentials.getAccessToken()));
+    String type = credentials.getTokenType();
+    if (!type.isEmpty()) {
```

Comment:
Can `type` be null?

Reply:
Not unless something went wrong but I add a check nonetheless